tools/integration_tests/multiwriters_and_multimounts/multiwriters_and_multimounts_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Provide integration tests for multi-writers writing to multiple GCSFuse mounts.
+package multiwriters_and_multimounts_test
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/mounting/multiple_mounting"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+const (
+	MountName1     = "mnt1"
+	MountName2     = "mnt2"
+	CommonFileName = "data.txt"
+)
+
+func TestMain(m *testing.M) {
+	setup.ParseSetUpFlags()
+
+	flags := [][]string{{"--implicit-dirs=false"}, {"--implicit-dirs=true"}}
+
+	if setup.TestBucket() == "" && setup.MountedDirectory() != "" {
+		log.Print("Please pass the name of bucket mounted at mountedDirectory to --testBucket flag.")
+		os.Exit(1)
+	}
+
+	// Requires two GCSFuse mount inside MountedDirectory named as MountName1 &
+	// MountName 2
+	setup.RunTestsForMountedDirectoryFlag(m)
```

Comment:
what tests are we running here. I thought we need to run tests only for multiple_mounts.

Reply:
This runs the same test but for already mountedDirectory. This is used in cases like GKE CSI driver where the mounting should be done outside of these tests.

Reply:
add a comment that we will exit here if mounted directory is present.